vm/test.c

```diff
@@ -190,9 +190,11 @@ main(int argc, char** argv)
         {.name = "register-offset", .val = 'r', .has_arg = 1},
         {.name = "unload", .val = 'U'}, /* for unit test only */
         {.name = "reload", .val = 'R'}, /* for unit test only */
+        {.name = "main-section", .val = 's', .has_arg = 1},
         {0}};
 
     const char* mem_filename = NULL;
+    const char* main_section_name = NULL;
```

Comment:
Instead of passing the section name, should we pass the name of the function to execute?

Reply:
We can *absolutely* do that.

Reply:
@Alan-Jowett I *believe* that the WIP I just pushed satisfies this requirement!